src/Storages/tests/gtest_columns_description_normalize.cpp

```diff
@@ -0,0 +1,27 @@
+#include <Storages/ColumnsDescription.h>
+#include <Common/tests/gtest_global_register.h>
+
+#include <gtest/gtest.h>
+
+using namespace DB;
+
+TEST(ColumnsDescription, Normalize)
+{
+    constexpr auto columns = R"(columns format version: 1
+3 columns:
+`a` UInt32
+`b` String	DEFAULT	If(a = 0, \'true\', \'false\')
+`c` String	DEFAULT	cAsT(a, 'String')
+)";
```

Comment:
I'd prefer to format such strings like this, but it's up to you.
```suggestion
    constexpr auto columns = "columns format version: 1\n"
                             "3 columns:\n"
                             "`a` UInt32\n"
                             "`b` String	DEFAULT	If(a = 0, \'true\', \'false\')\n"
                             "`c` String	DEFAULT	cAsT(a, 'String')\n";
```